test-app/src/main/ml-config/servers/java-functest.json

```diff
@@ -1,12 +1,13 @@
 {
-  "server-name": "java-functest",
-  "port": 8014,
-  "server-type": "http",
-  "root": "/",
-  "content-database": "java-functest",
-  "modules-database": "%%MODULES_DATABASE%%",
-  "default-error-format": "json",
-  "error-handler": "/MarkLogic/rest-api/error-handler.xqy",
-  "url-rewriter": "/MarkLogic/rest-api/rewriter.xml",
-  "rewrite-resolves-globally": true
+	"server-name": "java-functest",
+	"port": 8014,
+	"server-type": "http",
+	"root": "/",
+	"content-database": "java-functest",
+	"modules-database": "%%MODULES_DATABASE%%",
+	"default-error-format": "json",
+	"error-handler": "/MarkLogic/rest-api/error-handler.xqy",
+	"url-rewriter": "/MarkLogic/rest-api/rewriter.xml",
+	"rewrite-resolves-globally": true,
+	"authentication": "digestbasic"
```

Comment:
digestbasic is used now so that both digest and basic auth will work, as the reverse proxy only supports basic.